### ISSUE TITLE
Shotcut not start

### ISSUE THREAD

**Issue by dacrhu** [flathub/org.shotcut.Shotcut#58]
Hi,

Shotcut not start after upgrade.

OS: Fedora 35

App info:
Shotcut
org.shotcut.Shotcut
22.03.30
stable
flathub

I've attached console out.
Thank you!

[flatpak-console-out.txt](https://github.com/flathub/org.shotcut.Shotcut/files/8457749/flatpak-console-out.txt)

**Comment by hfiguiere** [comment]
Looks like it's crashing, but say nothing about it.

**Comment by ddennedy** [comment]
I had a problem too, but I could not trace it to Shotcut code but only something in Qt libs. Then, removing my saved settings fixed it. Shotcut saves the window size, position, and all panel layouts using 2 high level Qt functions; and it could be related to those.
Those settings are in `$HOME/.var/app/org.shotcut.Shotcut/config/Meltytech/Shotcut.conf` but I simply deleted `$HOME/.var/app/org.shotcut.Shotcut` since I did not care about proxies and custom presets.
